tests/keras_tests/test_keras_tp_model.py

```diff
@@ -235,12 +235,24 @@ def rep_data():
                                                                                   target_platform_capabilities=tpc)
 
     def test_get_keras_supported_version(self):
-        mct.get_target_platform_capabilities(TENSORFLOW, DEFAULT_TP_MODEL)
-        mct.get_target_platform_capabilities(TENSORFLOW, DEFAULT_TP_MODEL, "v1")
-        mct.get_target_platform_capabilities(TENSORFLOW, DEFAULT_TP_MODEL, "v2")
-        mct.get_target_platform_capabilities(TENSORFLOW, DEFAULT_TP_MODEL, "v3")
-        mct.get_target_platform_capabilities(TENSORFLOW, TFLITE_TP_MODEL, "v1")
-        mct.get_target_platform_capabilities(TENSORFLOW, QNNPACK_TP_MODEL, "v1")
+        tpc = mct.get_target_platform_capabilities(TENSORFLOW, DEFAULT_TP_MODEL) # Latest
+        self.assertTrue(tpc.version == 'v4')
```

Comment:
Maybe use constant for the versions' strings? Not mandatory just think that it would be easier to prevent future errors with all the code duplications we have here.

Reply:
You're right. It's worth considering enum for that, but let's do it in a different PR.